.github/workflows/ci.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       # if command with timeout already failed on unix, Windows job will be cancelled
       - run: pnpm test-plugin
         if: ${{ runner.os == 'Windows' }}
+      - uses: GabrielBB/xvfb-action@v1.6
+        with:
+          run: pnpm integration-test
```

Comment:
I reverted the commit where you deleted the integration tests from the pipeline (see 774358a042719f3b4f0ee6c1c16408b8c234c32f)

I think we should either delete integration tests entirely or add them to the pipeline.

Reply:
Done in 4dea1f10a56425407746e54ef1b677f21e319d21